### ISSUE TITLE
Drupal VM apache virtual host cannot find docroot

### ISSUE THREAD

**Issue by jezzaKratos** [acquia/blt#2445]
**My system information:**
* Operating system type: macOS High Sierra
* Operating system version: 10.13.2
* BLT version: 8.9.10 

**Output of `blt doctor`:**
```
$ blt doctor
Drupal VM was detected. Running blt doctor inside of VM...
Executing command cd /var/www/WebRedesign/docroot/.. && /var/www/WebRedesign/docroot/../vendor/bin/drush cc drush && /var/www/WebRedesign/docroot/../vendor/bin/drush --include=/var/www/WebRedesign/docroot/../vendor/acquia/blt/drush blt-doctor -r /var/www/WebRedesign/docroot/../docroot inside of Drupal VM...
'drush' cache was cleared.                                                                                                       [success]
+----------------------+----------------------------------------------------------------------------------+
| Property             | Value                                                                            |
+----------------------+----------------------------------------------------------------------------------+
| %paths               |                                                                                  |
|  - %root             | /var/www/WebRedesign/docroot                                                     |
|  - %site             | sites/default                                                                    |
|  - %modules          | modules                                                                          |
|  - %themes           | themes                                                                           |
|  - %config-sync      | /var/www/WebRedesign/config/default                                              |
|  - %files            | sites/default/files                                                              |
|  - %private          | /var/www/WebRedesign/files-private                                               |
|  - %temp             | /tmp                                                                             |
| admin-theme          | seven                                                                            |
| blt-version          | 8.9.10                                                                           |
| bootstrap            | Successful                                                                       |
| config-sync          | /var/www/WebRedesign/config/default                                              |
| db-driver            | mysql                                                                            |
| db-hostname          | localhost                                                                        |
| db-name              | drupal                                                                           |
| db-password          | drupal                                                                           |
| db-port              | 3306                                                                             |
| db-status            | Connected                                                                        |
| db-username          | drupal                                                                           |
| dev-desktop-enabled  | false                                                                            |
| drupal-settings-file | sites/default/settings.php                                                       |
| drupal-version       | 8.4.2                                                                            |
| drupal-vm-enabled    | true                                                                             |
| drush-alias-files    | /var/www/WebRedesign/docroot/../drush/site-aliases/example.local.aliases.drushrc |
|                      | .php                                                                             |
|                      | /var/www/WebRedesign/docroot/../drush/site-aliases/aliases.drushrc.php           |
|                      | /var/www/WebRedesign/docroot/../drush/site-aliases/example.acsf.aliases.drushrc. |
|                      | php                                                                              |
| drush-conf           | /var/www/WebRedesign/drush/drushrc.php                                           |
| drush-script         | /var/www/WebRedesign/vendor/drush/drush/drush.php                                |
| drush-temp           | /tmp                                                                             |
| drush-version        | 8.1.15                                                                           |
| files                | sites/default/files                                                              |
| install-profile      | lightning                                                                        |
| local-drushrc        | /var/www/WebRedesign/docroot/sites/default/local.drushrc.php                     |
| local-settings       | /var/www/WebRedesign/docroot/sites/default/settings/local.settings.php           |
| modules              | modules                                                                          |
| php-bin              | /usr/bin/php                                                                     |
| php-conf             | /etc/php/5.6/cli/php.ini                                                         |
| php-mysql            | /var/run/mysqld/mysqld.sock                                                      |
| php-os               | Linux                                                                            |
| private              | /var/www/WebRedesign/files-private                                               |
| root                 | /var/www/WebRedesign/docroot                                                     |
| shell                |                                                                                  |
| site                 | sites/default                                                                    |
| temp                 | /tmp                                                                             |
| theme                | stark                                                                            |
| themes               | themes                                                                           |
| uri                  | http://local.WebRedesign.com                                                     |
| user                 |                                                                                  |
+----------------------+----------------------------------------------------------------------------------+
+--------------------------+-------------------------------------------------------------+
| Check                    | Outcome                                                     |
+--------------------------+-------------------------------------------------------------+
| checkDocrootExists       | Found docroot.                                              |
| checkCoreExists          | Drupal core exists                                          |
| checkSettingsFile        | BLT settings are included in settings file.                 |
| checkLocalSettingsFile   | Found your local settings file.                             |
| checkLocalDrushFile      | Found your local drush settings file.                       |
| checkUri                 | $options['uri'] is set.                                     |
| checkUriResponse         | Received a response from site http://local.WebRedesign.com. |
| checkFileSystem:%files   | Public files directory is writable.                         |
| checkFileSystem:%private | Private files directory is writable.                        |
| checkFileSystem:%temp    | Temporary files directory is writable.                      |
| checkDbConnection        | Connected to database.                                      |
| checkDrupalBootstrapped  | Bootstrapped Drupal via drush.                              |
| checkDrupalInstalled     | Drupal is installed.                                        |
| checkCaching:page        | Drupal cache is disabled.                                   |
| checkCaching:css         | CSS preprocessing is disabled.                              |
| checkCaching:js          | JS preprocessing is disabled.                               |
| checkComposer:require    | acquia/blt is in composer.json's require object.            |
| checkBehat:exists        | Behat local settings file exists.                           |
| checkBehat:root          | Behat drupal_root is set correctly for Drupal VM.           |
| checkProjectYml:keys     | project.yml has no deprecated keys.                         |
| checkDrupalVm:alias      | drush.aliases.local exists your drush aliases file.         |
| checkDrupalVm            | Drupal VM is configured correctly.                          |
| checkPhpDateTimezone     | PHP setting for date.timezone is correctly set              |
+--------------------------+-------------------------------------------------------------+
Connection to 127.0.0.1 closed.
$
```

**When I run this command:**
Load local.(projectName).com on Google Chrome browser

**I get the following output:**
The browser redirects to 'http://www.local.(projectName).com'. The page displayed is the "Apache2 Ubuntu Default Page". The first paragraph reads, "_This is the default welcome page used to test the correct operation of the Apache2 server after installation on Ubuntu systems. It is based on the equivalent page on Debian, from which the Ubuntu Apache packaging is derived. If you can read this page, it means that the Apache HTTP server installed at this site is working properly. You should replace this file (located at /var/www/html/index.html) before continuing to operate your HTTP server._" 
**See screenshot image attached.**

**And I expected this to happen:**
I expected the BLT site to load, not the "Apache2 Ubuntu Default Page" to load instead. 

![2018-01-11_15-38-11-ubuntudefaultpage](https://user-images.githubusercontent.com/3794174/34850152-dcb15e00-f778-11e7-9b06-e6389a23ea31.png)

**Comment by grasmash** [comment]
This looks like a drupal vm issue. The virtual host is not correctly configured. Did you set up Drupal VM with `blt vm`? You can try re-creating it with `blt vm:nuke && blt vm`.

**Comment by jezzaKratos** [comment]
I did setup Drupal VM with blt vm. 

These were the steps I followed: 
Installed the following: Git, Composer, PHP, Vagrant, Virtualbox
1.            Checkout the source code from Bitbucket.
2.            Move into Code folder.
3.            Run command: composer update, composer install
4.            Install blt alias: composer run-script blt-alias
5.            Run command: blt vm
6.            Run command: blt setup
7.            Run command: blt doctor [there are no red highlighted issues]

I will try re-creating with the command you suggested, `blt vm:nuke && blt vm`
